sunpy/database/tables.py

```diff
@@ -596,7 +591,7 @@ def entries_from_fido_search_result(sr, default_waveunit=None):
 
 
 def entries_from_file(file, default_waveunit=None,
-                      time_string_parse_format=None):
+                      time_string_parse_format=''):
```

Comment:
Note: This was done so that `None` won't be passed to `Time.strptime` (which would make `strptime` freak out, if I remember correctly).